symengine/cwrapper.cpp

```diff
@@ -232,6 +232,12 @@ CWRAPPER_OUTPUT_TYPE symbol_set(basic s, const char *c)
     CWRAPPER_END
 }
 
+int number_is_zero(const basic s)
+{
+	SYMENGINE_ASSERT(is_a_Number(*(s->m)));
+	return (int)((down_cast<const Number &>(*(s->m))).is_zero());
```

Comment:
Can you change the two tabs starting the two lines above with 4 four space each?